app/assets/stylesheets/global.sass

```diff
@@ -67,6 +67,12 @@ html, body
       &:last-child:after
         content: none
 
+#availability
```

Comment:
This is never used (or at least I cannot find it in this PR 🙈 )

So can you please remove this?